packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -135,13 +135,12 @@ export const VerifyCodeDialog = ({
   );
 };
 
-export const CreateANewOrganizationForm = () => {
+export const CreateANewOrganizationForm = ({ slug }: { slug?: string }) => {
```

Comment:
Is this change required?

Reply:
We relied on router query params and now we get it through serverSideProp